common.gypi

```diff
@@ -268,6 +269,10 @@
     'msvs_cygwin_shell': 0, # prevent actions from trying to use cygwin
 
     'conditions': [
+      [ 'arm_version==7', {
+        # Any Cortex-A that can run V8 supports at least armv7-a+neon-vfpv3.
+        'cflags': [ '-march=armv7-a+neon-vfpv3' ],
```

Comment:
FWIW I did a brief search and at least Tegra 2-based (Cortex-A9 -- armv7-a) devices do not include neon.

Reply:
Yes, such SoCs exist but they aren't able to run V8 in ARMv7 mode, according to V8:

https://github.com/nodejs/node/blob/c849f2d4f8ff19c830b6a97bd6978bf41f878e8b/deps/v8/src/codegen/arm/assembler-arm.cc#L156-L163

https://github.com/nodejs/node/blob/c849f2d4f8ff19c830b6a97bd6978bf41f878e8b/deps/v8/src/codegen/arm/assembler-arm.cc#L218-L222

Tegra 2 specifically is pretty obsolete (we're talking 10 year old smartphones and tablets) but if someone wanted to build node for it, they can with `./configure -- -Darm_version=6`.

Is there a more recent SoC where this is an issue?